tests/views/test_message.py

```diff
@@ -212,14 +212,11 @@ def test_technical_inbox_threads_list(self, mock_request, mock_get_jwt):
     @requests_mock.mock()
```

Comment:
Ok, so you might need an approach change. When writing tests the first thing to do is make sure it's obvious the process you are testing and you do that with the name and assert. If the test gets complicated, you then you bring in Given, When, Then to help.
So whats this test for? Checking a link appears when a thread_id is present in the url, which unfortunately that name doesn't tell me that and the assert either. That assert string is far to ambiguous and definitely doesn't confirm there now is a link. We want tests as obvious as possible so when another dev comes along they read it and instantly now the action it is testing

Equally I am not a fan of the that jwt blah thing either, I know it's been used elsewhere but for me that's just proliferating rubbish. It doesn't have to 100% accurate the return and you don't need to change anywhere else, just try not to copy code like that